spec/jobs/subscription_confirm_job_spec.rb

```diff
@@ -271,4 +274,29 @@
       expect(mail_mock).to have_received(:deliver_now)
     end
   end
+
+  describe "#send_payment_authorization_emails" do
+    let(:order) { instance_double(Spree::Order) }
+    let(:mail_mock) { double(:mailer_mock, deliver: true) }
```

Comment:
Three checks `deliver` in this spec.

Reply:
Good catches, thanks! 980ac6ff2ddbc6d7767f80fb669dc8b77aa15aae